src/tokens/TokenImport.tsx

```diff
@@ -63,7 +63,6 @@ export default function TokenImportScreen(_: Props) {
   const { t } = useTranslation()
   const dispatch = useDispatch()
 
-  // Should I create a new config for this?
   const supportedNetworkIds = getSupportedNetworkIdsForTokenBalances()
```

Comment:
This should be good for now. We can enable/disable this feature by ff in any case.